messagebird/client.py

```diff
@@ -1,187 +1,255 @@
+
+    def group_update(self, id, name, params=None):
+        if params is None: params = {}
+        params.update({'name': name})
+        self.request_plain_text('groups/' + str(id), 'PATCH', params)
+
+    def group_add_contacts(self, groupId, contactIds):
+        query = self.__group_add_contacts_query(contactIds)
+        self.request_plain_text('groups/' + str(groupId) + '?' + query, 'PUT', None)
+
+    def __group_add_contacts_query(self, contactIds):
+        # __group_add_contacts_query gets a query string to add contacts to a
+        # group. The expected format is ids[]=first-contact&ids[]=second-contact.
+        # See: https://developers.messagebird.com/docs/groups#add-contact-to-group.
+        return '&'.join('ids[]=' + str(id) for id in contactIds)
+
+    def group_remove_contact(self, groupId, contactId):
+        self.request_plain_text('groups/' + str(groupId) + '/contacts/' + str(contactId), 'DELETE', None)
+
+    def conversation_list(self, limit=10, offset=0):
+        uri = CONVERSATION_PATH + '?' + self._format_query(limit, offset)
+        return ConversationList().load(self.request(uri, 'GET', None, CONVERSATION_TYPE))
+
+    def conversation_start(self, start_request):
+        uri = CONVERSATION_PATH + '/start'
+        return Conversation().load(self.request(uri, 'POST', start_request, CONVERSATION_TYPE))
+
+    def conversation_update(self, id, update_request):
+        uri = CONVERSATION_PATH + '/' + str(id)
+        return Conversation().load(self.request(uri, 'PATCH', update_request, CONVERSATION_TYPE))
+
+    def conversation_read(self, id):
+        uri = CONVERSATION_PATH + '/' + str(id)
+        return Conversation().load(self.request(uri, 'GET', None, CONVERSATION_TYPE))
+
+    def conversation_list_messages(self, conversation_id, limit=10, offset=0):
+        uri = CONVERSATION_PATH + '/' + str(conversation_id) + '/' + CONVERSATION_MESSAGES_PATH
+        uri += '?' + self._format_query(limit, offset)
+
+        return ConversationMessageList().load(self.request(uri, 'GET', None, CONVERSATION_TYPE))
+
+    def conversation_create_message(self, conversation_id, message_create_request):
+        uri = CONVERSATION_PATH + '/' + str(conversation_id) + '/' + CONVERSATION_MESSAGES_PATH
+        return ConversationMessage().load(self.request(uri, 'POST', message_create_request, CONVERSATION_TYPE))
+
+    def conversation_read_message(self, message_id):
+        uri = CONVERSATION_MESSAGES_PATH + '/' + str(message_id)
+        return ConversationMessage().load(self.request(uri, 'GET', None, CONVERSATION_TYPE))
+
+    def conversation_create_webhook(self, webhook_create_request):
+        return ConversationWebhook().load(
+            self.request(CONVERSATION_WEB_HOOKS_PATH, 'POST', webhook_create_request, CONVERSATION_TYPE))
+
+    def conversation_delete_webhook(self, id):
+        uri = CONVERSATION_WEB_HOOKS_PATH + '/' + str(id)
+        self.request(uri, 'DELETE', None, CONVERSATION_TYPE)
```

Comment:
no we are not, if you look in the client.py all the other methods are like that. I would also prefer to update this in another PR for all methods instead of in this one